kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/monitor/ModemMonitorServiceImpl.java

```diff
@@ -662,48 +656,42 @@ private void monitor() {
                                             timeTillReset);
                                 }
                             }
-                        } else if (pppState == PppState.CONNECTED) {
+                        } else if (pppSt == PppState.CONNECTED) {
                             this.resetTimerStart = System.currentTimeMillis();
                         }
 
-                        this.pppState = pppState;
+                        this.pppState = pppSt;
                         ConnectionInfo connInfo = new ConnectionInfoImpl(ifaceName);
                         InterfaceState interfaceState = new InterfaceState(ifaceName,
-                                LinuxNetworkUtil.hasAddress(ifaceName), pppState == PppState.CONNECTED,
+                                LinuxNetworkUtil.hasAddress(ifaceName), pppSt == PppState.CONNECTED,
                                 connInfo.getIpAddress());
                         newInterfaceStatuses.put(ifaceName, interfaceState);
                     }
 
-                    if (modem.isGpsSupported()) {
-                        if (isGpsEnabledInConfig(modem.getConfiguration())) {
-                            if (modem instanceof HspaCellularModem) {
-                                if (!modem.isGpsEnabled()) {
-                                    modem.enableGps();
-                                }
-                            }
-                            postModemGpsEvent(modem, true);
+                    if (modem.isGpsSupported() && isGpsEnabledInConfig(modem.getConfiguration())) {
+                        if ((modem instanceof HspaCellularModem) && !modem.isGpsEnabled()) {
```

Comment:
Could you please apply the cleanup? I believe that the inner parenthesis are not needed.